drift/core/beamtransfer.py

```diff
@@ -1435,8 +1435,6 @@ def project_vector_svd_to_sky(self, mi, vec, temponly=False, conj=False):
             SVD vector to return.
         """
         npol = 1 if temponly else self.telescope.num_pol_sky
-        print("temponly", temponly)
```

Comment:
I just noticed that `npol` is never actually used in this function: only `self.telescope.num_pol_sky` is used to set the shape of the output matrix. If we want to fix that in this PR, we should also check that it doesn't break anything else that expects `self.telescope.num_pol_sky` polarizations from this routine...

Reply:
it's used in the loop line 1455, it's okay to give it a shorter variable.

Reply:
its used in line 1455, where we loop over polarizations.

Reply:
Oops, you're right!